quality_control/models/qc_trigger_line.py

```diff
@@ -14,8 +14,12 @@ class QcTriggerLine(models.AbstractModel):
     user = fields.Many2one(
         comodel_name='res.users', string='Responsible',
         track_visibility='always', default=lambda self: self.env.user)
+    partner = fields.Many2many(
+        comodel_name='res.partner', string='partner',
+        help='If this field is empty it will consider for all the possible'
```

Comment:
If filled, the test will only be created when the action is done for one of the specified partners. If empty, the test will be always created.